### PR TITLE
Use new PG image for Help toolbar 

### PR DESCRIPTION
**Pull request by psteckler**

### CODE REVIEW

**Comment by psteckler** [comment]
The 6 commits shown here are a red herring. The only difference between this branch and the PG master branch should be the 2 help icon files.

File: images/epg-help.xpm [hunk 1]
```diff
@@ -1,248 +1,257 @@
 /* XPM */
-static char * epg_help_xpm[] = {
-"24 24 221 2",
+static char * help_xpm[] = {
```
Comment (cpitclaudel) [inline images/epg-help.xpm]: Should this be called epg_help_xmp intead of help_xmp? (Does it not make a difference?)

**Comment by cpitclaudel** [comment]
Thanks! I can just squash and merge to remove the spurious commits.

File: images/epg-help.xpm (hunk 1 above)
Comment (psteckler) [inline images/epg-help.xpm]: Probably it should be changed (not sure if it makes a difference). 

The name is an artifact of how I created the file.

Comment (psteckler) [inline images/epg-help.xpm]: I'll make the change in my branch and push.

Comment (psteckler) [inline images/epg-help.xpm]: Done.

**Comment by cpitclaudel** [comment]
Thanks! One more thing: the original was a square. Did that matter?

**Comment by psteckler** [comment]
Hmmm, I thought I'd asked gimp to make it 24x24. Probably when I set one dimension, it chose the other one. Let's see if I can fix that.

**Comment by psteckler** [comment]
Because the image was taller than wide, I cropped off the bottom to get a square image, then shrunk it down to 24x24, the old size.

**Pull request closed without merging**